plans/all.fmf

```diff
@@ -5,7 +5,8 @@ provision:
   # 120 means 120GB, disk has to be specificed here, the format from:
   #   https://tmt.readthedocs.io/en/stable/spec/hardware.html#disk
   # does not work here or below under "hardware"
-  disk: 120
+  disk:
```

Comment:
Oh, shouldn't this be under the hardware key? Look at the URL: https://tmt.readthedocs.io/en/stable/spec/hardware.html#disk